english/jumpsuit/jumpsuit-review.md

```diff
@@ -0,0 +1 @@
+That jumpsuit you're wearing looks stupid. It's right here in your [file](http://theportalwiki.com/wiki/GLaDOS)...
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.